README.md

```diff
@@ -570,6 +574,8 @@ parameters:
     address: https://localhost:8200
 ```
 
+Use `kapitan lint -h` to checkup on your inventory/secrets.
```

Comment:
why does it need `-h` after `lint` ?

Reply:
It doesn't, it's just pointing people to the lint --help to see all available options there. It felt weird copy pasting the options in the readme as well. I can add them if you think it's needed.

Reply:
I just found the sentence a bit confusing.

it could be either:

```
Use `kapitan lint -h` to view all the lint options
```

or

```
Use `kapitan lint` to checkup on your inventory/secrets.
```